jdaviz/core/template_mixin.py

```diff
@@ -1463,6 +1462,35 @@ def _sort_by_icon(items_dict):
 
         self._apply_default_selection()
 
+    def update_wcs_only_filter(self, wcs_only):
+        """
+        The layers that are populated in LayerSelect.choices
+        will be either WCS-only layers (for setting viewer orientation)
+        or non-WCS-only layers (for "real data"). This method toggles
+        the layer choices by adjusting the layer filters on this
+        LayerSelect instance.
+
+        Parameters
+        ----------
+        wcs_only : bool
+            `True` will filter only the WCS-only layers, `False` will
+            give the non-WCS-only layers.
+        """
+        def is_wcs_only(data):
```

Comment:
I feel like this re-implemented `layer_is_wcs_only` that you also implemented in Imviz helper.

Reply:
This needs to be a separate function that you can turn on and off as a `filter` for `LayerSelect`.